lib/LedgerSMB/Template/XLS.pm

```diff
@@ -65,8 +65,9 @@ my $rowcount;
 my $currcol;
 
 sub _worksheet_handler {
-    $rowcount = -1;
     $currcol = 0;
+    $rowcount = 0;
+    -- $rowcount;
```

Comment:
@ylavoie I think the code can be completely removed; the rowcount isn't used anywhere, is it?